src/components/modal/login/index.tsx

```diff
@@ -0,0 +1,60 @@
+import logo from '@/assets/favicon.ico';
+import googleLogo from '@/assets/googleLogo.png';
+import { Avatar, Heading, Icon, Modal, Text } from '@/components/common';
+import * as variants from '@/styles/variants.css';
+import { useGoogleLogin } from '@react-oauth/google';
+import { useNavigate } from 'react-router-dom';
+import * as style from './style.css';
```

Comment:
꼭 아래가 아니더라도 import문 정렬하면 좋을 것 같아요~
```suggestion
import { useNavigate } from 'react-router-dom';
import { Avatar, Heading, Icon, Modal, Text } from '@/components/common';
import { useGoogleLogin } from '@react-oauth/google';

import * as style from './style.css';
import * as variants from '@/styles/variants.css';
import logo from '@/assets/favicon.ico';
import googleLogo from '@/assets/googleLogo.png';
```